api/catalog/configuration/spectacular.py

```diff
@@ -0,0 +1,63 @@
+from textwrap import dedent
+
+from decouple import config
+
+
+SPECTACULAR_SETTINGS = {
+    "TITLE": "Openverse API",
+    "DESCRIPTION": dedent(
+        """
+        Openverse is a search engine for openly-licensed media. The Openverse
+        API is a system that allows programmatic access to public domain digital
+        media. It is our ambition to index and catalog billions of
+        openly-licensed works, including articles, songs, videos, photographs,
+        paintings, and more. Using this API, developers will be able to access
+        the digital commons in their own applications.
+        """
+    ),
+    "TOS": "https://docs.openverse.org/terms_of_service.html",
```

Comment:
Can we move the TOS to the API consumer docs somehow? Not necessarily in this PR.